app/views/sessions/new.html.haml

```diff
@@ -34,7 +34,7 @@
                          aria: {labelledby: "passwordLabel"}
 
     = f.hidden_field :remember_me, value: 1
-    = f.submit t('devise.sessions.new.sign_in'), class: 'btn btn-large btn-block btn-default'
+    = f.submit t('devise.sessions.new.sign_in'), class: 'btn btn-large btn-block btn-primary'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.